program/convert-inches-to-feet/ConvertInchesToFeet.js

```diff
@@ -0,0 +1,5 @@
+function inchesToFeet(inch){
+  return Math.round(inch/12);
+}
+
+inchesToFeet(60);
```

Comment:
Please add console.log()

Reply:
console.log is not required every-time , its a function with return value .